### ISSUE TITLE
Unable to reproduce the results

### ISSUE THREAD

**Issue by Arjung27** [gpcv-liujin/REDNet#7]
Hi! I downloaded the data using the link mentioned in the readme (13G almost) and tried running the code. I could not reproduce the numbers. Rather my numbers are way off. The training is highly unstable and the loss value is very noisy (not going down rather oscillating). I also tried adding RMVSNet to reproduce the numbers on that but wasn't able to do so. I think there is some issue with the data or data loader for WHU dataset. Could you or anyone please verify if you or they are able to reproduce the numbers using the repo?

**Comment by gpcv-liujin** [comment]
> 
> 
> Hi! I downloaded the data using the link mentioned in the readme (13G almost) and tried running the code. I could not reproduce the numbers. Rather my numbers are way off. The training is highly unstable and the loss value is very noisy (not going down rather oscillating). I also tried adding RMVSNet to reproduce the numbers on that but wasn't able to do so. I think there is some issue with the data or data loader for WHU dataset. Could you or anyone please verify if you or they are able to reproduce the numbers using the repo?

Sorry that we updated the camera parameters in the training data, but forgot to update the code at the same time. 
You need to replace the following code in " def tr_load_cam( )" of "preprocess.py" file :

    x0 = pera[0][1] - pera[0][7] - (pera[0][10] * pera[0][11])  # whu
    y0 = pera[0][2] - pera[0][8] - (pera[0][9] * pera[0][12])
To:

    x0 = pera[0][1] # whu
    y0 = pera[0][2]

The new version of the code has been re-uploaded.
Thank you very much for your feedback, and sorry again for causing trouble to you.

**Comment by Arjung27** [comment]
Thank you for clarifying. I will make that change. Give me a day to train the model and I will get back to you asap.
